tests/versioned/aws-server-stubs/response-server/index.js

```diff
@@ -26,6 +26,22 @@ function createResponseServer() {
     res.end()
   })
 
+  // server.destroy: close, but faster!
```

Comment:
it might not affect the current tests but i'll also add this to the empty-server-response server